configs/atmoz-sftp/atmoz-sftp.dockerfile

```diff
@@ -2,6 +2,10 @@
 # https://github.com/atmoz/sftp/issues/258
 FROM odidev/sftp
 
-COPY ./configs/atmoz-sftp/fix-permissions.sh /etc/sftp.d/fix-permissions.sh
+
+ARG SFTP_USERNAME
+ARG SFTP_PASSWORD
+RUN mkdir -p /etc/sftp
+RUN echo "$SFTP_USERNAME:$SFTP_PASSWORD:1001" > /etc/sftp/users.conf
```

Comment:
refactor to allow azure to use this.
The original wouldn't work as the CMD command uses something else in azure than locally, and thus the command entry point will not work. this allows us to configure it the same, without using CMD in any way